README.md

```diff
@@ -112,70 +117,112 @@ A `.env` file should be created in the repository root, to keep configuration de
 cp example.env .env
 ```
 
-Replace `REPO_PATH_SERVER` with the path from above.
+```
+cd workspace
+pwd
+cd ..
+```
+Please note the printing of the pwd command as `STABLE_ROOT_PATH`.
+
+Please adjust in the `.env` file at least the `REPO_PATH_SERVER` with the value from #1 and the `STABLE_ROOT_PATH` from the last command above.
+If you have apps you like to mount to all running nextcloud versions in this setup, just set the `ADDITIONAL_APPS_PATH`. 
+Note, this is not recommended for apps that have individual versions for different server versions. But that could be a good place to mount your apps you are working on to test them accross different server versions.
```

Comment:
`STABLE_ROOT_PATH` and `ADDITIONAL_APPS_PATH` should have a proper default value so on new installations this would no longer be needed and the values set during bootstrap.sh would work fine.

Reply:
(As introduced with 12d15a8c41e72d551c190bb3379c49b60bdafc2e)